Makefile

```diff
@@ -1,24 +1,20 @@
-CFLAGS += -DSHAPE -DCOLOR -Wall -pedantic -ansi -D_XOPEN_SOURCE
-LDLIBS = -lXext -lX11
-PREFIX ?= /usr
-BIN = $(DESTDIR)$(PREFIX)/bin
+include config.mk
 
-MANDIR = $(DESTDIR)$(PREFIX)/share/man/man1
-MANSUFFIX = 1
```

Comment:
> Honestly, I don't think 9wm has enough configuration here to require a separate file for this.

That's debatable due to new developers could easily associate `config.mk` with 9wm's compilation preferences.

>  Also I believe the include command is not portable to BSD make

Well, it's fairly used in BSD ports. Also, I just tested it in OpenBSD v7 — it failed. The headers are not properly specified.

See: https://mmap.page/dive-into/make

Reply:
Did some testing myself and it seems you're right include works on both. TIL!
It also looks like it will need some more modification to build on *BSD as you mentioned (and it definitely didn't work before either), I'll open an issue for that.